dotcom-rendering/index.d.ts

```diff
@@ -20,7 +20,8 @@ type ThemePillar =
 	| 'SportPillar'
 	| 'CulturePillar'
 	| 'LifestylePillar';
-type ThemeSpecial = 'SpecialReportTheme' | 'Labs';
+
+type ThemeSpecial = 'SpecialReportTheme' | 'Labs' | 'SpecialReportAltTheme';
```

Comment:
CAPI will be able to send this after [this](https://github.com/guardian/content-api-scala-client/pull/371) gets merged.

Reply:
Making a note for a future PR!